packages/stargate/src/testutils.spec.ts

```diff
@@ -1,9 +1,16 @@
+import { Random } from "@cosmjs/crypto";
+import { Bech32 } from "@cosmjs/encoding";
+
 export function pendingWithoutSimapp(): void {
   if (!process.env.SIMAPP_ENABLED) {
     return pending("Set SIMAPP_ENABLED to enable Simapp based tests");
   }
 }
 
+export function makeRandomAddress(): string {
+  return Bech32.encode("cosmos", Random.getBytes(20));
+}
```

Comment:
This is unused now. I think it would be nice to have something similar creating a random 20 byte array. The function would not do much but having the name is better than `Random.getBytes(20)`